api/server/routes_get.go

```diff
@@ -14,7 +14,10 @@ func (s *Server) handleRouteGet(c *gin.Context) {
 
 	appName := c.MustGet(api.AppName).(string)
 	routePath := path.Clean(c.MustGet(api.Path).(string))
-
+	//TODO does this validation make sense? move to validator?
+	// -removes trailing slash
+	// -permits non-rooted paths
+	// -permits '.', '..', ('../')*
```

Comment:
I'm thinking that ```datastoreutil.validator``` should have centralized/shared path validation, and maybe it makes sense to include cleaning there as well.  Part of validation would be to reject an un-cleaned path anyways.